addons/crm/models/crm_lead.py

```diff
@@ -764,6 +761,53 @@ def action_schedule_meeting(self):
         }
         return action
 
+    def action_send_mail(self):
+        self.ensure_one()
+        template = self.env.ref('crm.email_template_opportunity_mail')
```

Comment:
Can't you re-use action_lead_mass_mailand tweak it, like explained for sms sending ? Seems easier.